docs/ops/sequence/CTCGreedyDecoderV2_6.md

```diff
@@ -0,0 +1,69 @@
+## CTCGreedyDecoderV2 <a name="CTCGreedyDecoderV2"></a> {#openvino_docs_ops_sequence_CTCGreedyDecoderV2_6}
```

Comment:
We didn't add the version to the operation name. Why do you add V2 to the operation name?
@lazarevevgeny Is it correct?

Reply:
Yes, you are right. We already provided this feedback online but Ilya have not yet updated it:)